.ci/schedule-weekly.groovy

```diff
@@ -44,7 +45,13 @@ pipeline {
     stage('Top failing Beats tests - last 7 days') {
       steps {
         setEnvVar('YYYY_MM_DD', new Date().format("yyyy-MM-dd", TimeZone.getTimeZone('UTC')))
-        runWatcherForBranch(branches: ['main', '8.<minor>', '8.<next-patch>', '7.<minor>'])
+        runWatcherForBranch(project: 'beats', branches: ['main', '8.<minor>', '8.<next-patch>', '7.<minor>'], to: env.BEATS_MAILING_LIST)
+      }
+    }
+    stage('Top failing Elastic Agent tests - last 7 days') {
```

Comment:
Would it be possible to send this very same message for the top failing E2E tests?

Reply:
yep